joomla-projects/gsoc21_media-manager
# Week 2: Generate responsive images everywhere, remove them if needed

Pull Request for issues #6 and #7, #11.

### Summary of Changes
- Implemented **automatic deletion** of responsive images when they are no longer needed: when they get removed from the content or media manager.
- Added responsive image generation to **every content editor and form** where it's needed by using plugins and helper functions.


### Testing Instructions
Go to the categories, banners, contacts, articles, or newsfeeds pages and add an image to the content editor or form. Then check out the responsive directory created in the folder where the original image is stored, responsive versions will be there. Now delete the original image, responsive versions will be gone.
